tests/curriculum_to_html_test.py

```diff
@@ -0,0 +1,7 @@
+from adam.curriculum.phase1_curriculum import EACH_OBJECT_BY_ITSELF_SUB_CURRICULUM
+from adam.curriculum_to_html import CurriculumToHtml
+
+
+def test_simple_curriculum_html():
```

Comment:
For the test, write your output to a temporary directory: http://doc.pytest.org/en/latest/tmpdir.html